projects/fsastorefrontstyles/scss/base/forms.scss

```diff
@@ -12,14 +12,3 @@
   font-weight: var(--cx-font-weight-semi);
   text-transform: uppercase;
 }
-
-.custom-file-input {
-  cursor: pointer;
-  &:focus ~ .custom-file-label {
-    border-color: var(--cx-color-secondary);
-    box-shadow: none;
-  }
-}
-*[role='button'] {
```

Comment:
This selector was also affecting maps indicator on Find an agent page, so it's also moved